src/NuGet.Clients/NuGet.VisualStudio.Implementation/Extensibility/VsPackageInstaller.cs

```diff
@@ -269,7 +269,7 @@ public void InstallPackagesFromVSExtensionRepository(string extensionId, bool is
 
                     var projectContext = new VSAPIProjectContext(skipAssemblyReferences, disableBindingRedirects);
 
-                    return InstallInternalAsync(
+                    await InstallInternalAsync(
```

Comment:
I generally like the syntax of using `await` instead of returning the Task directly when in a long delegate like this, but given we're in escrow, I would revert this too, for the reasons given above.